hist/hist/src/TH1.cxx

```diff
@@ -2675,7 +2688,29 @@ TObject* TH1::Clone(const char* newname) const
       // when dictionary information is initialized, so we need to
       // keep obj->fFunction valid during its execution and
       // protect the update with the write lock.
+      auto oldstats = fFunctions->FindObject("stats");
+
+      // Avoid duplication of this histogram
+      static TClassRef clStats("TPaveStats");
+      static auto setParent = GetSetParentFunc(clStats);
```

Comment:
libHist seems to not depend on libGraf. Are we guaranteed that we can find `SetParent` the first time we try to initialize the static?

Reply:
right .. actually not ... hence the attractiveness of your option/proposal [Albeit we can make it work by putting the static inside the ```if (oldstats)``` (and propagating them forward via a regular variable).